internal/datacoord/compaction.go

```diff
@@ -39,7 +39,6 @@ import (
 	"github.com/milvus-io/milvus/pkg/util/typeutil"
 )
 
-//go:generate mockery --name=compactionPlanContext --structname=MockCompactionPlanContext --output=./  --filename=mock_compaction_plan_context.go --with-expecter --inpackage
```

Comment:
keep this.
you can modify this in next pr.

Reply:
updated